L1Trigger/L1TTrackMatch/plugins/L1TrackerEtMissEmulatorProducer.cc

```diff
@@ -338,15 +338,15 @@ void L1TrackerEtMissEmulatorProducer::produce(edm::Event& iEvent, const edm::Eve
     edm::LogVerbatim("L1TrackerEtMissEmulatorProducer")
         << "====MET===\n"
         << "Integer MET: " << EtMiss.Et << "| Integer MET phi: " << EtMiss.Phi << "\n"
-        << "Float MET: " << (EtMiss.Et) * l1tmetemu::kStepMET
+        << "Float MET: " << EtMiss.Et.to_double() * l1tmetemu::kStepMET
```

Comment:
There is no need to multiply by `l1tmetemu::kStepMET` as the "step" width is already accounted for by the `ap_ufixed`.